c2rust-transpile/src/translator/assembly.rs

```diff
@@ -239,6 +368,35 @@ fn rewrite_reserved_reg_operands(att_syntax: bool, operands: &mut Vec<BidirAsmOp
     (prolog, epilog)
 }
 
+/// Removes comments from an x86 assembly template.
+fn remove_comments(mut asm: &str) -> String {
```

Comment:
Could we keep the comments and prepend them to the Rust assembly statement?

Reply:
The comment-stripped asm is only used for the "AT&T or Intel" heuristic. I can add a comment to clarify that.